packages/discord.js/typings/index.d.ts

```diff
@@ -4224,10 +4233,10 @@ export interface AddOrRemoveGuildMemberRoleOptions {
   reason?: string;
 }
 
-export class GuildMemberManager extends CachedManager<Snowflake, GuildMember, UserResolvable> {
+export class GuildMemberManager extends CachedManager<Snowflake, GuildMember | PartialGuildMember, UserResolvable> {
```

Comment:
`PartialGuildMember` should never be added to cache, that's the main purpose of this PR after all.